packages/api/src/sequelize/migrations/2023-12-14_00_create-document-retrieval-result.ts

```diff
@@ -0,0 +1,47 @@
+import { DataTypes, literal } from "sequelize";
+import type { Migration } from "..";
+
+const tableName = "document_retrieval_result";
+
+// Use 'Promise.all' when changes are independent of each other
+export const up: Migration = async ({ context: queryInterface }) => {
+  await queryInterface.sequelize.transaction(async transaction => {
+    await queryInterface.createTable(
```

Comment:
given that the interface definitions might change and evolve as we discover more about the possible response types we might recieve + us just having an oversight since its pretty complicated, would it be better to treat the whole thing as a JSONB the way we deal with external Data for patients so that we can minimize the number of migrations we have to do. Especially since the schema is not validated yet and has already changed a few times

Reply:
nevermind thats literally what you have already 😂